setup.py

```diff
@@ -213,7 +213,7 @@
       version='1.0.1',
       packages=['slimish_jinja'],
       scripts=['slim_to_jinja.py'],
-      install_requires=['distribute', 'jinja2'],
+      install_requires=['setuptools', 'jinja2'],
```

Comment:
setuptools isnt needed here.

`install_requires` is for packages which the code depends on.  after install, the code doesnt invoke on `setuptools`.

e.g. `uncompyle6` should be added here if you are using it, but IMO that isnt a good idea.